### PR TITLE
Import classes and modules in __init__.py of each package

### PR DESCRIPTION
**Pull request by ghost**
Classes and modules defined in the TensorFlow tree were imported in the file `__init__.py` of each package to make the current package-level imports shorter.

### CODE REVIEW

**Review by eric-heiden — APPROVED** [review]

File: sandbox/rocky/tf/algos/__init__.py [hunk 1]
```diff
@@ -1 +1,4 @@
-
+from .batch_polopt import BatchPolopt
```
Comment (ryanjulian) [inline sandbox/rocky/tf/algos/__init__.py]: https://www.python.org/dev/peps/pep-0008/#id23

Please avoid relative imports

File: examples/trpo_gym_tf_cartpole.py [hunk 2]
```diff
@@ -34,9 +33,4 @@
     # optimizer=ConjugateGradientOptimizer(hvp_approach=FiniteDifferenceHvp(base_eps=1e-5))
 )
 
-run_experiment_lite(
-    algo.train(),
-    n_parallel=1,
-    snapshot_mode="last",
-    seed=1
-)
+run_experiment_lite(algo.train(), n_parallel=1, snapshot_mode="last", seed=1)
```
Comment (ryanjulian) [inline examples/trpo_gym_tf_cartpole.py]: This is ugly but probably what was demanded by YAPF. It's definitely better to have the list style.

You can disable YAPF when it is wrong using this comment.
```python
run_experiment_lite(
    algo.train(),
    n_parallel=1,
    snapshot_mode="last",
    seed=1
) # yapf: disable
```

File: sandbox/rocky/tf/core/__init__.py [hunk 3]
```diff
@@ -1 +1,7 @@
-
+from .parameterized import Parameterized
```
Comment (ryanjulian) [inline sandbox/rocky/tf/core/__init__.py]: no relative imports

File: sandbox/rocky/tf/distributions/__init__.py [hunk 4]
```diff
@@ -1 +1,6 @@
-
+from .bernoulli import Bernoulli
```
Comment (ryanjulian) [inline sandbox/rocky/tf/distributions/__init__.py]: no relative imports

File: sandbox/rocky/tf/envs/__init__.py [hunk 5]
```diff
@@ -1 +1,3 @@
-
+from .base import TfEnv
```
Comment (ryanjulian) [inline sandbox/rocky/tf/envs/__init__.py]: no relative imports

File: sandbox/rocky/tf/launchers/vpg_cartpole.py [hunk 6]
```diff
@@ -25,10 +24,5 @@
     max_path_length=100,
     n_itr=40,
     discount=0.99,
-    optimizer_args=dict(
-        tf_optimizer_args=dict(
-            learning_rate=0.01,
-        )
-    )
-)
+    optimizer_args=dict(tf_optimizer_args=dict(learning_rate=0.01, )))
```
Comment (ryanjulian) [inline sandbox/rocky/tf/launchers/vpg_cartpole.py]: # yapf: disable

File: sandbox/rocky/tf/optimizers/__init__.py [hunk 7]
```diff
@@ -1 +1,5 @@
-
+from .conjugate_gradient_optimizer import ConjugateGradientOptimizer
```
Comment (ryanjulian) [inline sandbox/rocky/tf/optimizers/__init__.py]: no relative imports

File: sandbox/rocky/tf/policies/__init__.py [hunk 8]
```diff
@@ -1 +1,6 @@
-
+from .base import Policy
```
Comment (ryanjulian) [inline sandbox/rocky/tf/policies/__init__.py]: no relative imports

File: sandbox/rocky/tf/q_functions/__init__.py [hunk 9]
```diff
@@ -0,0 +1 @@
+from .base import QFunction
```
Comment (ryanjulian) [inline sandbox/rocky/tf/q_functions/__init__.py]: no relative imports

File: sandbox/rocky/tf/samplers/__init__.py [hunk 10]
```diff
@@ -1 +1,2 @@
-
+from .batch_sampler import BatchSampler
```
Comment (ryanjulian) [inline sandbox/rocky/tf/samplers/__init__.py]: no relative imports

**Review by ryanjulian — CHANGES_REQUESTED** [review]
Great work, this definitely makes the whole codebase more readable with one small change.

Some comments:
1. __Remove the YAPF formatting__
I apologize for not anticipating this earlier. The intention of the policy of only reformatting edited files is to gradually reformat the codebase while we maintain it, where maintain is defined as "touching a file." A large-scale change like this, which touches every file in a small way, violates the spirit of the rule. It implicitly makes this a (reformat everything + change imports) PR, but we only want the PR to do one thing. __So for this change, please only change the import lines and don't bother with YAPF or PEP8 for the files.__ In the future we might just push one giant PR which YAPFifys the whole codebase. We will ignore the CI and push the change through. Again, I apologize for the inconvenience.

1. __Do it for the whole codebase, not just the TF tree__
It's important to be consistent. We are maintaining all of rllab, even though we focus on TF mostly.

1. __No relative imports__
PEP8 recommends not using relative imports except in special circumstances. The logic behind this is that relative imports obscure the absolute package path when a non-developer is browsing the repository, and they also make bugs from refactors and renamings harder to find. I don't think this circumstance is special enough. https://www.python.org/dev/peps/pep-0008/#id23

**Comment by ghost** [comment]
@ryanjulian I reverted the changes concerning PEP8 formatting and changed the relative imports for absolute imports. Also I had to merge one commit from the integration branch.
Please review the changes again.

**Review by ryanjulian — DISMISSED** [review]
LGTM

**Comment by ryanjulian** [comment]
I was just about to merge this, but realized it's only for the TF tree. Can you do it for *all* of rllab?

**Comment by ghost** [comment]
@ryanjulian the change requested is done, please review.
Again, the CI build is failing due to check_yapf.sh:
image

**Comment by ryanjulian** [comment]
The latest run seems to be executing fine.

https://travis-ci.org/ryanjulian/rllab/builds/382443801?utm_source=github_status&utm_medium=notification

**Review by ryanjulian — APPROVED** [review]

**Comment by ryanjulian** [comment]
Fixes #66

**Pull request merged**